content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -70,6 +70,39 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 
 {{< /note >}}
 
+### Enable dual-stack via kubeadm
```

Comment:
i'm not sure if this is the best place for this, actually.

maybe this page can just cross link to a new section under the https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/

sig-docs will likely provide feedback on this.

Reply:
my vote here is to move this as a new page under:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/
and cross reference to this dual-stack.md page.

but we should wait for sig-docs feedback.

Reply:
> wait for sig-docs feedback

OK

Reply:
For SIG Docs usually try _not_ to put task descriptions in the concepts section. We'd prefer to either create a new task page or add a section to an existing task page.

Sometimes https://kubernetes.io/docs/setup/ is a better home; that's usually for documents that only apply to brand new clusters.